tsconfig.json

```diff
@@ -1,16 +1,21 @@
 {
+    "$schema": "https://json.schemastore.org/tsconfig",
     "compilerOptions": {
-        "target": "es5",
+        "target": "esnext",
         "module": "commonjs",
-        "declaration": true,
+        "moduleResolution": "node",
```

Comment:
Why not directly esnext module?

Reply:
Because it is a small but breaking change rather than a refactoring

Reply:
Because it is a small but breaking change rather than a refactoring